src/modules/claim/claim.controller.ts

```diff
@@ -134,14 +140,20 @@ export class ClaimController {
 
     await validateOrReject(claimDTO);
 
-    const payload = JSON.stringify(claimData);
-    this.logger.debug(`publishing claims request ${payload} from ${did}`);
-    data.claimIssuer.map(issuerDid => {
-      this.nats.connection.publish(
-        `${issuerDid}.${NATS_EXCHANGE_TOPIC}`,
-        payload,
-      );
-    });
+    const result = await this.claimService.handleClaimEnrollmentRequest(
```

Comment:
Minor point, but please rename `handleClaimEnrollmentRequest` to `handleClaimEnrolmentRequest` as this matches the naming convention used across the IAM stack
```suggestion
    const result = await this.claimService.handleClaimEnrolmentRequest(
```